fixtures/ci-teleport-rbac/ci-teleport.yaml

```diff
@@ -6,30 +6,24 @@ metadata:
   name: ci-teleport
 rules:
 - apiGroups: [""]
-  resources: ["users", "groups", "serviceaccounts"]
+  resources: ["users", "groups"]
```

Comment:
Would this work here? Not sure if you can use `resourceNames` with the `impersonate` verb.

```yaml
rules:
- apiGroups: [""]
  resources: ["users"]
  verbs: ["impersonate"]
- apiGroups: [""]
  resources: ["groups"]
  verbs: ["impersonate"]
  resourceNames: ["teleport-ci-test-group"]
```

Reply:
I tried that on GKE and minikube and it doesn't work unfortunately.